public/javascript/script.js

```diff
@@ -40,40 +40,39 @@
     saveInputArray.forEach((input) => {
         if(input.index === id) {
             input.index = id;
-            input.social_id = socialId;
-            input.social_name = socialName;
-            input.social_image = socialImage;
-            input.social_meta = socialMeta;
+            input.socialId = socialId;
+            input.socialName = socialName;
+            input.socialImage = socialImage;
+            input.socialMeta = socialMeta;
             edited = true;
         }
-    })
+    });
 
     // if not edited create one
     if(!edited) {
         saveInputArray.push({ 
                     "index": id,
                     "id": socialId,
-                    "social_name" : socialName,
-                    "social_image": socialImage,
-                    "social_meta": socialMeta
+                    "socialName" : socialName,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/Monsapps/myBlog/pullRequest?prid=8352134)
```suggestion
                    socialName,
```